news/2022-09-27-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,153 @@
+
+**Strain** is a concept across all game modes that considers how many times and for how long a player is subjected to high-intensity sections within a beatmap. Sections with very high speed, difficulty or other pertaining factors will significantly increase a beatmap's considered strain values.
+
+We will also be using terminology coined by the osu!taiko community to refer to how patterns are placed. **d** refers to don (red) notes, and **k** refers to kat (blue) notes. Repeated notes of the same colour are referred to as *monos* (short for monocolour).
+
+If you would like to know more about the changes posted, or for a more detailed breakdown on the some of the in-development changes, head on over to the [osu!taiko rework document](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit). This is a working document, and will periodically be updated with the latest developments from any developers who request their changes to be added to it, via the osu!taiko pp committee.
+
+## An insight into the base of the new rework
+
+The last set of changes made to osu!taiko's performance points & star rating algorithm aimed to chart several skills at once. This not only made it difficult to collect meaningful and targeted feedback relevant to one specific skill, but also led to increased complexity which halted continuous development efforts. 
+
+As such, we have opted to take a more modular approach by introducing new changes incrementally, in the hopes of being able to fix any reported issues one step at a time. Moving forward, osu!taiko star rating will be receiving continuous updates, potentially placing star rating and performance points in a state of flux, especially over the coming few months. 
+
+During this time, we will be working on other difficulty components (both small and large-scale) in preparation for future releases. It is our hope that this will continue to help the algorithm reflect the latest trends and expectation in difficulty calculation, but we also apologise in any advance for any disruption this may cause as we begin working things out.
+
+In the star rating part of this rework, we have essentially completely reworked the stamina and colour skills, introducing new concepts and the infrastructure required to support them.
+
+Performance points have seen significant changes to mod multipliers and a broad change to how accuracy is calculated. Please note however that these changes are *experimental* and did not receive as much scrutiny as the rest of the rework. Consider leaving feedback if you find anything that feels especially unusual.
```

Comment:
This seems to be in the wrong place. Move it down to be part of the performance section?

Reply:
Also I dunno about "experimental". LIke that sounds like nothing has been reviewed and we're just going gung-ho. Hopefully that's not the case, because it seems amateur.

Reply:
just like star rating, this introduction section also introduces the pp changes, so it's probably in the right place,